packages/core/src/header-builder/authorization-header.ts

```diff
@@ -5,9 +5,8 @@ import {
   DestinationAuthToken,
   getOAuth2ClientCredentialsToken,
   sanitizeDestination
-} from '../scp-cf';
```

Comment:
Don't the files in `header-builder` belong into one of the other packages?

Reply:
yes I move them around as I think they belong to.